src/components/Modal.js

```diff
@@ -19,6 +19,12 @@ export default class Modal {
     }
   };
 
+  setLoading(isLoading) {
+    isLoading
+      ? (this._saveButton.textContent = "Saving...")
+      : (this._saveButton.textContent = "Saved.");
```

Comment:
`"Saved.`


You need to return the initial button text back to the button. So, it should be `Save`, `Create` and so on